src/ape_cache/_cli.py

```diff
@@ -0,0 +1,53 @@
+import click
+import pandas as pd
+
+from ape import networks
+from ape.cli import NetworkBoundCommand, network_option
+from ape.logging import logger
+from ape.utils import ManagerAccessMixin
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Command-line helper for managing query caching database.
```

Comment:
Optional suggestion:

```suggestion
    Manage query caching database.
```

because I think "Command-line help" is implied enough.